native/src/components/MapView.tsx

```diff
@@ -85,35 +93,30 @@ const MapView = ({
   bottomSheetFullscreen,
   zoom,
 }: MapViewProps): ReactElement => {
-  const cameraRef = useRef<MapLibreGL.Camera>(null)
-  const mapRef = useRef<MapLibreGL.MapView>(null)
+  const cameraRef = useRef<CameraRef>(null)
+  const mapRef = useRef<MapViewRef>(null)
   const [followUserLocation, setFollowUserLocation] = useState<boolean>(false)
   const { refreshPermissionAndLocation } = useUserLocation({ requestPermissionInitially: true })
   const { t } = useTranslation('pois')
   const theme = useTheme()
 
-  const bounds = {
-    ne: [boundingBox[2], boundingBox[3]],
-    sw: [boundingBox[0], boundingBox[1]],
-  }
-
   const coordinates = selectedFeature?.geometry.coordinates
   const defaultZoom = coordinates ? normalDetailZoom : defaultViewportConfig.zoom
-  const defaultSettings: CameraSettings = {
+
+  const [cameraSettings, setCameraSettings] = useState({
     zoomLevel: zoom ?? defaultZoom,
-    centerCoordinate: coordinates,
-    bounds: coordinates ? undefined : bounds,
-  }
+    centerCoordinate: coordinates ?? cityCoordinates,
```

Comment:
❓ Changing from a bounding box to just center coordinates might lead to problems with zooming as there are regions with different sizes. Are bounding boxes not supported anymore?

Reply:
Ah, good point, I don't remember anymore why I changed it, and it still works, so I'll go back to the previous version 😅